.github/workflows/CodeAnalysisAndCI.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         working-directory: gaas-ui/
 
     steps:
+      - uses: actions/checkout@v1
```

Comment:
Is there a reason not to use `actions/checkout@v2`? Think it is more maintained.

Reply:
Good spot have changed this to v2.